aviary/subsystems/aerodynamics/flops_based/solved_alpha_group.py

```diff
@@ -5,7 +5,7 @@
 
 import aviary.constants as constants
 from aviary.subsystems.aerodynamics.aero_common import DynamicPressure
-from aviary.subsystems.aerodynamics.gasp_based.table_based import CruiseAero as TabularCruiseAero
+from aviary.subsystems.aerodynamics.gasp_based.table_based import CruiseAeroUsingTable as TabularCruiseAero
```

Comment:
Suggestion to rename the class to `TabularCruiseAero` instead of aliasing the name? It would be simpler and shorter